spec/ameba/rule/lint/unused_block_argument_spec.cr

```diff
@@ -0,0 +1,110 @@
+require "../../../spec_helper"
+
+module Ameba::Rule::Lint
+  subject = UnusedBlockArgument.new
+
+  describe UnusedBlockArgument do
+    it "doesn't report if it is an instance var argument" do
+      s = Source.new %(
+        class A
+          def initialize(&@callback)
+          end
+        end
+      )
+      subject.catch(s).should be_valid
+    end
+
+    it "doesn't report if anonymous" do
+      s = Source.new %(
+        def method(a, b, c, &)
+        end
+      )
+      subject.catch(s).should be_valid
```

Comment:
shouldn't we use that newer matcher `expect_no_issues`/`expect_issue` instead?

Reply:
yeah, we should but I was too lazy to rewrite those, xmas & all that jazz, sorry!

Reply:
There are still many specs to rewrite anyway...